src/common/tests_lib.c

```diff
@@ -355,3 +356,73 @@ dts_sgl_init_with_strings_repeat(d_sg_list_t *sgl, uint32_t repeat,
 	v_dts_sgl_init_with_strings_repeat(sgl, repeat, count, d, valist);
 	va_end(valist);
 }
+
+/* dmg cmd wrappers for daos_test */
+static int
+daos_dmg_fork_exec(char *argv[])
+{
+        pid_t pid = fork();
+
+        if (pid == -1) {
+                D_ERROR("fork() failed to create child\n");
+                return -DER_INVAL;
+        } else if (pid == 0) {
+                /* child process */
+                D_INFO("child pid = %d\n", getpid());
+                execvp(argv[0], argv);
+		D_ERROR("execvp %s failed\n", argv[0]);
+                exit(-DER_INVAL);
+        } else {
+	        /* parent process */
+        	D_INFO("parent pid = %d\n", getpid());
+
+	        int status;
+
+        	if (waitpid(pid, &status, 0) == -1) {
+                	D_ERROR("waitpid() failed for pid = %d\n", pid);
+                	return -DER_INVAL;
+        	}
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs